lms/djangoapps/instructor/tests/test_legacy_xss.py

```diff
@@ -39,6 +39,9 @@ def setUp(self):
             course_id=self._course.id
         )
 
+    def _is_sudo(self, region=None):    # pylint: disable=unused-argument
```

Comment:
Why is this single-use function declared as a method? Could you re-define it as an inline lambda? It's confusing for this named method to always return `True`.